cmake/DD4hepConfig.cmake.in

```diff
@@ -25,9 +25,17 @@ set ( DD4HEP_USE_CXX14   "@DD4HEP_USE_CXX14@" )
 set ( Geant4_DIR         "@Geant4_DIR@" )
 set ( GEANT4_USE_CLHEP   "@GEANT4_USE_CLHEP@" )
 
-set ( ROOTSYS            "$ENV{ROOTSYS}" )
-set ( ROOT_DIR           "$ENV{ROOTSYS}/cmake" )
-set ( ROOT_VERSION       "@ROOT_VERSION@" )
+
+if(NOT ENV{ROOTSYS})
```

Comment:
Sorry, yes you are right.  This might not be the most appropriate commit. But here is the reason for doing this. Using `ROOTSYS` (which is really a legacy env variable) in the build will avoid problems on systems where it is not used. For example https://github.com/AIDASoft/DD4hep/issues/162. Ideally only the information in `ROOTConfig.cmake` would be used for finding ROOT dependencies.